publishedbranches/docs-commandline-tools.yaml

```diff
@@ -0,0 +1,15 @@
+prefix: '/'
```

Comment:
I believe this should be 'tools'

Reply:
or 'database-tools' (depending on which site hosts the content of this repo) -- but certainly not '/' :)

Reply:
oh lord, of course. cloud-docs's prefix is `/` because it's at its own weird url, right?

Reply:
yes exactly!!